### PR TITLE
Add a parsely_post_authors filter Add a parsely_post_authors filter

### PR DESCRIPTION
**Pull request by adamsilverstein**
Allows themes to extend the reported authors, for example adding authors from a custom taxonomy.

### CODE REVIEW

**Comment by msukmanowsky** [comment]
@adamsilverstein ditto here, can we move the filter to before the `get_clean_parsely_page_value` call for the values in the array?

**Comment by adamsilverstein** [comment]
Sure, makes sense. I will resubmit.

**Comment by adamsilverstein** [comment]
Ok, updated this as well. Here is an example of how we used it, to pull in our custon person taxonomy... feel free to use or link this in your docs:

https://gist.github.com/adamsilverstein/1c1291440f935faab085

**Comment by adamsilverstein** [comment]
@msukmanowsky ^^ updated :)

File: wp-parsely.php [hunk 1]
```diff
@@ -686,6 +686,8 @@ private function get_author_names($post) {
             $authors = array(get_user_by('id', $post->post_author));
         }
         $authors = array_map(array($this, 'get_author_name'), $authors);
+        $authors = array_map(array($this, 'get_author_name'), $authors);
```
Comment (msukmanowsky) [inline wp-parsely.php]: Whoops, looks like we now have a duplicated `$authors = array_map(array($this, 'get_author_name'), $authors);` line @adamsilverstein.

**Comment by adamsilverstein** [comment]
@msukmanowsky Oops! Fixed.

**Comment by msukmanowsky** [comment]
Perfect, thanks!

**Pull request merged**